core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -1302,10 +1307,7 @@ static void filterJarFile(Path resolvedDep, Path targetPath, Set<String> transfo
                             entry.setCompressedSize(-1);
                             out.putNextEntry(entry);
                             try (InputStream inStream = in.getInputStream(entry)) {
-                                int r = 0;
```

Comment:
This is intentional and we have tested before, reusing the buffer is faster in this case

Reply:
Ok thanks